01/app.js

```diff
@@ -1 +1,10 @@
-console.log('DOM');
\ No newline at end of file
+console.log("DOM");
+const comment = document.querySelector(
+  ".comments__item, .comments__item--newest"
```

Comment:
Prawidłowe rozwiązanie to `.comments__item.comments__item--newest` to co napisałaś wyżej oznacza, że szukamy albo jednej klasy albo drugiej więc to działa, ale nie spełnia założeń zadania.